lib/taskana-core-test/src/test/java/acceptance/task/create/CreateTaskWithAttachmentAccTest.java

```diff
@@ -0,0 +1,201 @@
+import pro.taskana.classification.api.exceptions.ClassificationNotFoundException;
+import pro.taskana.classification.api.models.Classification;
+import pro.taskana.classification.api.models.ClassificationSummary;
+import pro.taskana.common.api.exceptions.InvalidArgumentException;
+import pro.taskana.common.internal.util.Pair;
+import pro.taskana.common.internal.util.Quadruple;
+import pro.taskana.task.api.TaskService;
+import pro.taskana.task.api.models.Attachment;
+import pro.taskana.task.api.models.ObjectReference;
+import pro.taskana.task.api.models.Task;
+import pro.taskana.testapi.TaskanaInject;
+import pro.taskana.testapi.TaskanaIntegrationTest;
+import pro.taskana.testapi.builder.TaskAttachmentBuilder;
+import pro.taskana.testapi.builder.TaskBuilder;
+import pro.taskana.testapi.builder.WorkbasketAccessItemBuilder;
+import pro.taskana.testapi.security.WithAccessId;
+import pro.taskana.workbasket.api.WorkbasketPermission;
+import pro.taskana.workbasket.api.WorkbasketService;
+import pro.taskana.workbasket.api.models.WorkbasketSummary;
+
+/**
+ * Acceptance test for all "create task" scenarios that involve {@linkplain
+ * pro.taskana.task.api.models.Attachment Attachments}.
+ */
+@TaskanaIntegrationTest
+class CreateTaskWithAttachmentAccTest {
+  @TaskanaInject TaskService taskService;
+  @TaskanaInject WorkbasketService workbasketService;
+  @TaskanaInject ClassificationService classificationService;
+
+  ClassificationSummary defaultClassificationSummary;
+  WorkbasketSummary defaultWorkbasketSummary;
+  ObjectReference defaultObjectReference;
+  Attachment defaultAttachment;
+  Task defaultTaskWithAttachment;
+
+  @WithAccessId(user = "businessadmin")
+  @BeforeAll
+  void setup() throws Exception {
+    defaultClassificationSummary =
+        defaultTestClassification()
+            .serviceLevel("P2D")
+            .buildAndStoreAsSummary(classificationService);
+    defaultWorkbasketSummary = defaultTestWorkbasket().buildAndStoreAsSummary(workbasketService);
+
+    WorkbasketAccessItemBuilder.newWorkbasketAccessItem()
+        .workbasketId(defaultWorkbasketSummary.getId())
+        .accessId("user-1-1")
+        .permission(WorkbasketPermission.OPEN)
+        .permission(WorkbasketPermission.READ)
+        .permission(WorkbasketPermission.APPEND)
+        .buildAndStore(workbasketService);
+    defaultObjectReference = defaultTestObjectReference().build();
+    defaultAttachment =
+        TaskAttachmentBuilder.newAttachment()
+            .classificationSummary(defaultClassificationSummary)
+            .objectReference(defaultObjectReference)
+            .build();
+    defaultTaskWithAttachment =
+        TaskBuilder.newTask()
+            .primaryObjRef(defaultObjectReference)
+            .workbasketSummary(defaultWorkbasketSummary)
+            .classificationSummary(defaultClassificationSummary)
+            .attachments(defaultAttachment)
+            .buildAndStore(taskService, "user-1-1");
```

Comment:
please be consistent and static import all test-api classes :)

btw: should we make that a guideline?

Reply:
I will include this in the guidelines :)

Reply:
What do you think about static imports of DefaultTestEntities instead of all test-api classes?